packages/aws-cdk-lib/aws-lambda/lib/function.ts

```diff
@@ -536,15 +536,29 @@ export interface FunctionOptions extends EventInvokeConfigOptions {
 
   /**
    * Sets the application log level for the function.
+   * @deprecated Use `applicationLogLevelV2` as a property instead.
```

Comment:
can you add this tag to `logFormat` as well? Looks like we forgot to add it in this PR #28942